Gemfile

```diff
@@ -37,6 +37,8 @@ gem 'rails_12factor', group: :production
 gem 'globalize'
 gem 'globalize-accessors'
 
+gem 'friendly_id'
```

Comment:
This is a good opportunity to try the behavior of both solutions when using Globalize. If they both work without creating a mess of config, we'll have to make a decision based on:

- We should use the same strategy for all models, so what is the effort to implement it?
- How do we manage uniqueness? (that one is very easy but both solutions have completely different solutions. with to_param, we should raise a validation error and change the value manually by resubmitting the form. With friendly_id, we can define the rules to generate a slug without requiring any other action from the user)